lib/maxitest/autorun.rb

```diff
@@ -1,4 +1,15 @@
 require "minitest"
+
+# Set ENABLE_PLUGINS before minitest/autorun so our plugin can check it
+module Maxitest
+  ENABLE_PLUGINS = true
+end
+
+# MT6 doesn't auto-load plugins, so we need to load them manually
+if Minitest::VERSION >= "6"
+  Minitest.load_plugins
+end
```

Comment:
do you have thoughts on not doing this, so that if someone uses minitest 6 they have to load plugins manually as intended ?